crc_errors/CRC_Count_check.py

```diff
@@ -11,11 +11,8 @@
 from ats.log.utils import banner
```

Comment:
new import style is `from pyats.log.utils import banner`

eg , rename ats -> pyats